### PR TITLE
PIC-4008 enable jira integration when deploying

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ orbs:
   owasp: entur/owasp@0.0.10
   hmpps: ministryofjustice/hmpps@8
   slack: circleci/slack@4.1.1
-  jira: circleci/jira@1.2.2
```
Comment (markreesmoj) [inline .circleci/config.yml:7]: This is already included by the hmpps orb

File: .circleci/config.yml [hunk 2]
```diff
@@ -53,25 +52,21 @@ workflows:
...
               only:
                 - main
                 - /.*deployable_branch.*/
-          post-steps:
-            - jira/notify:
-              job_type: build
       - hmpps/deploy_env:
           name: deploy_dev
           env: "dev"
+          jira_update: true
```
Comment (markreesmoj) [inline .circleci/config.yml:69]: will show in JIRA which environment the commit has been deployed to if the message is prefixed with the ticket number, like this commit

**Review by johnyfarrar — APPROVED** [review]
Great work - should we replicate this across all services?

**Comment by markreesmoj** [comment]
This was already working - just bringing it up to date with the latest orb

**Pull request merged**